data.sql

```diff
@@ -35,4 +35,17 @@ INSERT INTO animals(
 VALUES(
 	'Devimon', '2017-05-12',
 	'5', 'yes', '11'
-)
\ No newline at end of file
+)
+
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Plantmon', '2021-11-15', -5.7, 'no', 2);
```

Comment:
* The query for this is missing `Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.`
* The value for neutered for animal `Plantmon` should be `yes` not `no` 
* Below is a screenshot of the highlighted requirements that are missing kindly write queries to answer these conditions in your `queries.sql` file 
![image](https://user-images.githubusercontent.com/52497246/228217906-a0b63f95-c0b1-4c87-9fc9-339dff87aa6f.png)